core/src/main/java/org/infinispan/functional/impl/FunctionalMapImpl.java

```diff
@@ -41,7 +41,7 @@
 
    private static <K, V> long getFlagsBitSet(Cache<K, V> cache) {
       long flagsBitSet = 0;
-      for (;;) {
+      for (; ; ) {
```

Comment:
I wish we could use the good ol' `#define EVER ;;` and type `for(EVER) { ... }` :)